app/app.go

```diff
@@ -755,6 +756,7 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	paramsKeeper.Subspace(ibchost.ModuleName)
 	paramsKeeper.Subspace(icacontrollertypes.SubModuleName)
 	paramsKeeper.Subspace(icahosttypes.SubModuleName)
+	paramsKeeper.Subspace(icaauthmoduletypes.ModuleName)
 
```

Comment:
I guess the upgrade handler should set this param to something?

Reply:
There are two ways a doing this AFAIK.

- Disable `InitGenesis` of a module and set params manually.
- Let the default params set by `InitGenesis`.

For interchain accounts, we are manually setting params (and disabling `InitGenesis`) because we want to modify the default params. But for `icaauth` we don't want to modify the default value of genesis params.

Reply:
I've added a test for this in `test_upgrade.py` by querying params after the upgrade.

Reply:
ok